doc/reference/orm.rst

```diff
@@ -359,14 +348,14 @@ For the various field types and parameters, see :ref:`the fields reference
 
 Default values are defined as parameters on fields, either a value::
 
-    a_field = fields.Char(default="a value")
+    name = fields.Char(default="a value")
```

Comment:
watchout the name clash between `_name` and `name`. For a complete beginner it can be confusing

Reply:
+1, could create confusion.